src/services/offer.js

```diff
@@ -35,7 +35,9 @@ class OfferService {
         requirements,
     }) {
 
-        const ownerName = (await Company.findById(owner)).name;
+        const company = await Company.findById(owner);
+        const ownerName = company.name;
+        const ownerLogo = company.logo;
```

Comment:
```suggestion
        const {ownerName, ownerLogo} = await Company.findById(owner);
```

Reply:
Does this work since *Company* does not have any attribute named `ownerName` and `ownerLogo`?

Reply:
Sorry, ahahah
```suggestion
        const {name: ownerName, logo: ownerLogo} = await Company.findById(owner);
```

Reply:
Addressed requested changes